whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapRelationService.java

```diff
@@ -66,6 +68,21 @@ public List<String> getDomainsByRelationType(final String pkey, final RelationTy
                 .toList();
     }
 
+
+    public void includeRirSearchConformance(final RdapObject rdapObject, final String requestUrl){
+        rdapObject.getRdapConformance().add(RdapConformance.RIR_SEARCH_1.getValue());
+        if (requestUrl == null) {
+            return;
+        }
+        if (requestUrl.contains(RdapRequestType.IPS.name().toLowerCase())){
+            rdapObject.getRdapConformance().addAll(List.of(RdapConformance.IPS.getValue(), RdapConformance.IP_SEARCH_RESULTS.getValue()));
+            return;
```

Comment:
I do not think you can add IP_SEARCH_RESULTS and AUTNUM_SEARCH_RESULTS by default. It should be added only if it is an array.

Reply:
Discussed over huddle, section 6 points out this scenario